math_long.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
The short solution obviously better. Clear and readable code.

But in real development, depending on the situation, sometimes a `long` solution can be better. Such understanding becomes only through practice.